OJP/OJP_Trips.xsd

```diff
@@ -674,12 +674,12 @@
 		</xs:annotation>
 		<xs:sequence>
 			<xs:choice>
-				<xs:element name="TransferMode" type="TransferModesEnumeration">
+				<xs:element name="TransferType" type="TransferTypeEnumeration">
```

Comment:
TransferTypeEnumeration includes parkAndRide and BikeAndRide, how does this work?

Reply:
I think that having to park something takes additional time. That was the idea. You disagree?

Reply:
I don't disagree with the additional time. P&R and B&R in my understanding is that I drive my own car/bike to a parking lot connected to a stop, leave it there, and then transfer to public transit. So it would only make sense as a first mile. How does it work as a TransferLeg?

Reply:
@herlitze  wouldn't you  model  ?? as a TransferLeg?   car -> car parking at station --> ?? --> station    . Especially if it is a park garage or something like that. Pls advise. and then I update the model